xml/admin_operating_pools.xml

```diff
@@ -64,825 +65,831 @@
-   that will be used with &cephfs;, or pools that are automatically created by
-   &ogw; are automatically associated.
+   A pool can be created as either <literal>replicated</literal> to recover from lost OSDs by
+   keeping multiple copies of the objects or <literal>erasure</literal> to have
+   generalized RAID5 or 6 capability. Replicated pools require more raw storage,
+   while erasure coded pools require less raw storage. The default setting is
+   <literal>replicated</literal>. For more information on erasure coded pools, see
+   <xref linkend="cha-ceph-erasure"/>.
   </para>
-
   <para>
-   For other cases, you can manually associate a free-form application name
-   with a pool:
+   To create a replicated pool, execute:
   </para>
-
-<screen>&prompt.cephuser;ceph osd pool application enable <replaceable>pool_name</replaceable> <replaceable>application_name</replaceable></screen>
-
-  <tip>
-   <title>Default application names</title>
-   <para>
-    &cephfs; uses the application name <literal>cephfs</literal>, &rbd; uses
-    <literal>rbd</literal>, and &ogw; uses <literal>rgw</literal>.
-   </para>
-  </tip>
-
+<screen>&prompt.cephuser;ceph osd pool create <replaceable>POOL_NAME</replaceable> <replaceable>PG_NUM</replaceable> <replaceable>PGP_NUM</replaceable> replicated <replaceable>CRUSH_RULESET_NAME</replaceable> \
+<replaceable>EXPECTED_NUM_OBJECTS</replaceable></screen>
   <para>
-   A pool can be associated with multiple applications, and each application
-   can have its own metadata. You can display the application metadata for a
-   given pool using the following command:
+   To create an erasure coded pool, execute:
   </para>
-
-<screen>&prompt.cephuser;ceph osd pool application get <replaceable>pool_name</replaceable></screen>
- </sect1>
- <sect1 xml:id="ceph-pools-operate">
-  <title>Operating pools</title>
-
+<screen>&prompt.cephuser;ceph osd pool create <replaceable>POOL_NAME</replaceable></screen>
+  <note>
+    <para>
+      The autoscaler will take care of the remaining optional arguments. For
+      more information, see <xref linkend="op-pgs-autoscaler"/>.
+    </para>
+  </note>
   <para>
-   This section introduces practical information to perform basic tasks with
-   pools. You can find out how to list, create, and delete pools, as well as
-   show pool statistics or manage snapshots of a pool.
+   The <command>ceph osd pool create</command> command can fail if you exceed the
+   limit of placement groups per OSD. The limit is set with the option
+   <option>mon_max_pg_per_osd</option>.
   </para>
+  <variablelist>
+   <varlistentry>
+    <term>POOL_NAME</term>
+    <listitem>
+     <para>
+      The name of the pool. It must be unique. This option is required.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>PG_NUM</term>
+    <listitem>
+     <para>
+      The total number of placement groups for the pool. This option is
+      required. Default value is 8.
```

Comment:
As of SES7, this option is **NOT** required because the autoscaler is active by default.

Look:

```
master:~ # ceph osd pool create foo
pool 'foo' created
```

Also, I suggest to move the new paragraph (the one that starts with "In previous versions of SES, it was necessary to set the number of placement groups on each pool explicitly. . . .") here.